### PR TITLE
RFC 36: wpt.live

### PR DESCRIPTION
**Pull request by jugglinmike**

### CODE REVIEW

File: rfcs/wpt_live.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+contributors to preview their submissions during the peer review process,
+hosted at http://wptpr.live.
+
+## Details
+
+Bocoup and Google have collaborated on a service to provide a
+publicly-accessible instance of the web-platform-tests. The source code and the
+infrastructure configuration are maintained in the following repository:
+
+https://github.com/bocoup/web-platform-tests.live
```
Comment (jgraham) [inline rfcs/wpt_live.md]: Is there a plan to move that under the wpt repo?

In general I'd like to see documentation here about what additional non-code pieces are in play for this new service (credentials, certtificates, etc.). It needs to be possible for people to get the access required to perform ongoing maintainance.

Comment (jugglinmike) [reply rfcs/wpt_live.md]: > Is there a plan to move that under the wpt repo?

Yup; I've amended the RFC to state that

> In general I'd like to see documentation here about what additional non-code pieces are in play for this new service (credentials, certtificates, etc.). It needs to be possible for people to get the access required to perform ongoing maintainance.

Just access to the "wpt-live" project in Google Cloud Platform. I believe this is the extent of the secrets necessary to administer the project. Certificate renewal is automated via Let's Encrypt, so it doesn't represent a source of privileged information. I've updated the "risks" section with this information.

**Comment by foolip** [comment]
@web-platform-tests/wpt-core-team a week has passed now. Per our process it would be OK to merge this now, but positive feedback from someone who hasn't previously been involved with this work would be great!

**Comment by jdm** [comment]
This looks like a sensible way to address the issues with w3c-test.org and mitigate the risks around doing so. Exciting!

**Comment by foolip** [comment]
Alrighty, thanks all, I feel comfortable merging this now.

**Comment by foolip** [comment]
@jugglinmike has transferred the repo now and I've renamed it: https://github.com/web-platform-tests/wpt.live

File: rfcs/wpt_live.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+benefits by continuing to rely on that service. However, the service lacks
+automated failure recovery mechanisms, and its unreliability is well-known by
+both its users and its maintainers. Its status as a W3C-owned project reduces
+the ability of WPT members to understand and fix problems.
+
+Inherit the existing solution - To the latter point, another alternative is for
+WPT to inherit w3c-test.org. However, because this would not improve
+reliability, doing so would almost increase the maintenance effort required
+from WPT project members. (If this proposal is accepted, the maintainers of
+w3c-test.org will configure that domain to redirect to wpt.live).
```
Comment (Ms2ger) [inline rfcs/wpt_live.md]: @sideshowbarker sounds like it's time to do this.

**Pull request merged**